src/inmanta/server/services/compilerservice.py

```diff
@@ -369,13 +371,22 @@ async def start(self) -> None:
         await self._recover()
         self.schedule(self._cleanup, opt.server_cleanup_compiler_reports_interval.get(), initial_delay=0)
 
+    async def prestop(self) -> None:
+        await super(CompilerService, self).prestop()
+
+    async def stop(self) -> None:
+        await super(CompilerService, self).stop()
+
```

Comment:
Why override these explicitly? I'm not against it, just curious.

Reply:
For no specific reason actually. I am going to remove them. It's quite useless.